app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java

```diff
@@ -38,16 +38,16 @@
     private String sessionTitle;
 
     @ColorRes
-    private int sessionVividColorResId;
+    private int sessionVividColorResId = TopicColor.NONE.vividColorResId;
```

Comment:
> If topic color is null it is crash.

I guess it doesn't occur crash because `TopicColor.from` method returns NONE when topic is null.

```java
    public static TopicColor from(@Nullable Topic topic) {
        if (topic == null) {
            return NONE;
```

So maybe it's not necessary to set default value.